manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -66,7 +74,7 @@ func readContentFromFile(filename string) string {
 	return contents
 }
 
-func NewHttpdConfigMap(cr *miqv1alpha1.ManageIQ) *corev1.ConfigMap {
+func NewHttpdConfigMap(cr *miqv1alpha1.ManageIQ) (*corev1.ConfigMap, func() error) {
```

Comment:
I think `func() error` should probably be `controllerutil.MutateFn` as that type is defined and exported.